Documentation/network/kubernetes/ipam.rst

```diff
@@ -10,10 +10,19 @@
 Configuring IPAM Modes
 **********************
 
+Cilium supports multiple IP Address Management (IPAM) modes to meet the needs of different environments and cloud providers. 
+
+The following sections provide documentation for each supported IPAM mode:
+
 .. toctree::
    :maxdepth: 1
-   :glob:
 
-   ipam-crd
-   ipam-cluster-pool
-   ipam-multi-pool
+   ../concepts/ipam/cluster-pool
+   ../concepts/ipam/kubernetes
+   ../concepts/ipam/multi-pool
+   ../concepts/ipam/azure
+   ../concepts/ipam/azure-delegated-ipam
+   ../concepts/ipam/eni
+   ../concepts/ipam/gke
+   ../concepts/ipam/crd
+   ../concepts/ipam/deep_dive
```

Comment:
GKE is just `kubernetes` mode. CRD is already linked above. Deep dive is not an IPAM mode.

```suggestion
```